assets/css/lesson-editor.scss

```diff
@@ -0,0 +1,34 @@
+@import '~@wordpress/base-styles/breakpoints';
+
+#content-drip-upgrade {
+	.inside {
+		display: flex;
+		flex-direction: column;
+
+		@media (min-width: $break-medium)  {
```

Comment:
This will not work when you move the element  (without resizing the screen) for example if you move it down not to be on the side but on the bottom. 

I'm trying to figure out how to fix that for my task. Will post it here :D

Reply:
Totally forgot that this could go in the sidebar, thanks. I've fixed it in 49ab545a2c92aa2f72273ad09c5cfcac8f5ee795. Let me know if you find a better solution.

Reply:
Just curious as to where `$break-medium` is defined? I can't find it anywhere.

Reply:
Hey thats perfect solution :D Just one thing you should actually remove **flex-direction: column;**   now that you have wrap because if you leave it it looks like this in mobile: 

<img width="455" alt="image" src="https://user-images.githubusercontent.com/7208249/154200632-9a05e4ab-4546-4c91-950c-53b777993109.png">

But if you remove it looks like this: 
<img width="449" alt="image" src="https://user-images.githubusercontent.com/7208249/154200668-0d042143-c90b-45ef-afa6-2df6be428c41.png">

Reply:
You are totally right - fixed in 01915a6d5b7efd1704a3f2304099d8c57854a14c.